apps/console/src/extensions/components/logs/components/infinite-scroll-container.tsx

```diff
@@ -88,13 +112,34 @@ const InfiniteScrollContainer = (props: InfiniteScrollContainerPropsInterface):
         setActiveIndex(tempIndexArr);
     };
 
+    function handleLogDataViewClose(): void {
+        setView(false);
+        setCurrentLog(null);
+    }
```

Comment:
Please follow the arrow function declarations.
```suggestion
    const handleLogDataViewClose = (): void => {
        setView(false);
        setCurrentLog(null);
    }
```